app/src/androidTest/java/com/github/se/project/ui/profile/CreateTutorScheduleTest.kt

```diff
@@ -72,24 +65,5 @@ class CreateTutorScheduleTest {
   @Test
   fun availabilityScreen_displaysProfileNameAndInstructions() {
     composeTestRule.setContent { CreateTutorSchedule(navigationActions, listProfilesViewModel) }
-
-    // Verify that the greeting text is displayed correctly
```

Comment:
Not sure to understand this removal neither? As nothing is left or almost nothing is left of the test. What's the purpose of keeping those few lines if they test nothing?

Reply:
This test will be fixe by Alexis and was identified last week as flaw test, it was added on the scrumboard to be fixed by him, no worries here

Reply:
Okay, in this case I would suggest to improve you PR description for better understanding for the reviewer and the others that migth read it.